src/webserver.cpp

```diff
@@ -46,17 +46,17 @@ static void handle_sql(struct mg_connection* nc, struct http_message* hm)
 
     json::Object result;
     result[L"query"] = json::String(wxString(query).ToStdWstring());
-    //bool ret = Model_Report::instance().get_objects_from_sql(wxString(query), result); 
+    Model_Report::instance().get_objects_from_sql(wxString(query), result); 
 
-    for (const auto & r : Model_Setting::instance().all())
+    /*for (const auto & r : Model_Setting::instance().all())
```

Comment:
Output all setting but not sql query result has looks like test debug solution.
I've commented out settings output and uncommented sql query output.